public_html/wp-content/plugins/wordcamp-speaker-feedback/wordcamp-speaker-feedback.php

```diff
@@ -105,7 +105,19 @@ function add_feedback_page() {
 
 	$organizer_note  = '<!-- wp:paragraph {"textColor":"white","customBackgroundColor":"#94240b"} -->';
 	$organizer_note .= '<p style="background-color:#94240b" class="has-text-color has-background has-white-color">';
-	$organizer_note .= __( 'This page is a placeholder for the Speaker Feedback form. The content here will not be shown on the site.', 'wordcamporg' );
+	$organizer_note .= __( 'Organizer Note: This page is used to display the session list for Speaker Feedback form. It will be added after the content you enter here. You can remove this note.', 'wordcamporg' );
```

Comment:
```suggestion
	$organizer_note .= __( 'Organizer Note: This page is used to display the session list for the Speaker Feedback form. It will be added after the content you enter here. You can remove this note.', 'wordcamporg' );
```